Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -1125,7 +1126,13 @@ def search(user_id, subject='', _from='', to='', before='', after='', filename='
         'v1',
         ['https://www.googleapis.com/auth/gmail.readonly'],
         command_args['userId'])
-    result = service.users().messages().list(**command_args).execute()
+    try:
+        result = service.users().messages().list(**command_args).execute()
+    except HTTPError as e:
```

Comment:
I think that this way we would miss all of the mesasges.

Reply:
I would suggest try getting the messages per user, and skip the users without a mail service.

Reply:
this function searches in a specific mailbox, there is an iteration on all mailboxes in the calling function.